input_generator/raw_dataset.py

```diff
@@ -309,11 +325,28 @@ def process_coords_forces(
             )
             return
         else:
-            cg_coords, cg_forces, force_map = slice_coord_forces(
-                coords, forces, self.cg_map, mapping, force_stride, batch_size
-            )
+            if filter_cis:
+                coords, forces = filter_cis_frames(
+                    coords, forces, topology, verbose=True
+                )
+            if coords.shape[0] != 0:
+
+                # since the cis-pro filtering might have removed a lot of frames
+                # we need to make sure the force_stride is not too large
+                # ie there are at least min(n_frames, 100) frames left after striding
+                while coords.shape[0]<100*force_stride:
+                    force_stride /= 10
```

Comment:
This currently breaks the 1L2Y example because the force_stride goes from int to float

```suggestion
                    force_stride =  force_stride // 10
```